frontend/src/__tests__/cypress/cypress/fixtures/e2e/dataScienceProjects/testDeployKserveRaw.yaml

```
@@ -0,0 +1,5 @@
+projectDisplayName: KServe Raw Test Project
+projectDescription: Test project for KServe Raw deployment mode testing
+projectResourceName: test-kserve-raw-project
+singleModelName: 'test-model'
+modelOpenVinoPath: kserve-openvino-test/openvino-example-model
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)